layers/+tools/pdf-tools/packages.el

```diff
@@ -0,0 +1,114 @@
+;;; packages.el --- pdf-tools Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: André Peric Tavares <andre.peric.tavares@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq pdf-tools-packages
+      '(
+        pdf-tools
+        ))
+
+(defun pdf-tools/init-pdf-tools ()
+  (use-package pdf-tools
+    :defer t
+    :mode (("\\.pdf$" . pdf-view-mode))
+    :config
+    (progn
+      (pdf-tools-install)
+      ;; TODO: Make `/', `?' and `n' work like in Evil
+      (evilified-state-evilify pdf-view-mode pdf-view-mode-map
+        ;; Navigation
+        "j"  'pdf-view-next-line-or-next-page
+        "k"  'pdf-view-previous-line-or-previous-page
+        "l"  'pdf-view-next-page-command
+        "h"  'pdf-view-previous-page-command
```

Comment:
Right, I agree with `h` and `l`, it feels more consistent.

Now, for the upper case ones, I suggest this:
J -> Scroll page down
K -> Scroll page up
H -> Go to previous page
L -> Go to next page

H and L represent left and right, so it's like turning a real page.
I think J and K should scroll down/up because this is probably the most frequent action when reading a PDF. Normally we'd do this with `C-d` and `C-u` but this is cumbersome IMO.

Thanks for the suggestions.